packages/orbit-components/src/Tile/components/TileHeader/index.tsx

```diff
@@ -39,10 +39,10 @@ const IconRight = ({
   expandable?: Props["expandable"];
   className?: string;
 }) => {
-  if (expandable) return <ChevronDown color="secondary" className={className} />;
-  if (external) return <NewWindow className={className} />;
+  if (expandable) return <ChevronDown color="secondary" className={className} ariaHidden />;
+  if (external) return <NewWindow className={className} ariaHidden />;
```

Comment:
Is it a good idea to hide this icon? Best practice would be to use ariaLabel here and say that it's being opened in a new window. Should we add a new prop for the teams to be able to do that?

Reply:
Hmm that raises a bigger question: we are always rendering a div, and we allow defining the role. So screen readers can announce that it is a link. But it will never have an href… which is weird. Because it should be the link responsible to saying that it's external and opens a new window, not the icon…

I'd say, if anything, this is done in a separate task, and not here.

Reply:
Please create a task and / or raise this on the next team sync 🙏